### ISSUE TITLE
Support for iOS

### ISSUE THREAD

**Issue by Soleil-des-chats** [nillerusr/source-engine#374]
Maybe it's difficult but i think porting Source Engine is actually possible, for example Source 2 has an iOS version [(Dota Underlords)](https://developer.valvesoftware.com/wiki/Dota_Underlords).

**Comment by ironmjp** [comment]
I wonder if the mac build with a few tweak would run on ios

**Comment by mlshv** [comment]
I was wondering the same thing over the past few days. While I’m not familiar with the project, I’d love to help if someone has a vision of how it can be done

**Comment by ironmjp** [comment]
if you ported the mac version to ios then added the android buttons....

**Comment by ironmjp** [comment]
If we can compile for Mac arm then you have the processor sorted out. a base model iPhone allocates around 2gb to an app so source could run beautifully. and Mac OS and iOS have so similar structures that iOS apps can run on Macs no problem so the only problems are things that Mac OS supports but iOS doesn't but it shouldn't be that big.

**Comment by GuestSneezeOSDev** [comment]
![a](https://media.discordapp.net/attachments/856675014085836831/1093633793203638363/1027410C-0761-4EC6-8CBD-842EA77948E4.gif?ex=6763d997&is=67628817&hm=8e202e5a2ca028ada59d08eb2cf0864cb9162e9274cfc9b235d7fe96bb332af0&)

**Comment by ItzVladik** [comment]
No, source don't support IOS, but it is possible to make port, but nobody
want do it sadly

вс, 13 апр. 2025 г., 14:25 Mohamed Ashraf ***@***.***>:

**Comment by ItzVladik** [comment]
IOS dont have OpenGL Support anymore

вс, 13 апр. 2025 г., 15:06 Mohamed Ashraf ***@***.***>:

**Comment by ItzVladik** [comment]
*Metal

вс, 13 апр. 2025 г., 16:00 Mohamed Ashraf ***@***.***>:

**Comment by ironmjp** [comment]
no opengl works on ios but isnt updated. same as mac

**Comment by pullobi** [comment]
I'm gonna look into this and try to get something working

**Comment by Soleil-des-chats** [comment]
i'm already doing it now ;w;
